client/state/google-apps-users/reducer.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External Dependencies
+ */
+
+import { combineReducers } from 'redux';
+import uniqBy from 'lodash/uniqBy';
+import property from 'lodash/property';
+
+/**
+ * Internal Dependencies
+ */
+import {
+	GOOGLE_APPS_USERS_FETCH,
+	GOOGLE_APPS_USERS_FETCH_COMPLETED,
+	GOOGLE_APPS_USERS_FETCH_FAILED,
+	SERIALIZE,
+	DESERIALIZE
+} from 'state/action-types';
+
+export function items( state = [], action ) {
+	switch ( action.type ) {
+		case GOOGLE_APPS_USERS_FETCH_COMPLETED:
+			return uniqBy( state.concat( action.items ), property( 'email' ) );
```

Comment:
Is there a need to use `property( 'email' )` ? Is it not sufficient to call with as `uniqBy( state.concat( action.items ), 'email' );` ?